fe/fe-core/src/main/java/com/starrocks/sql/optimizer/rule/mv/MaterializedViewRule.java

```diff
@@ -134,9 +146,11 @@ public static boolean isExistMVs(OptExpression root) {
         checkCompensatingPredicates(columnNameToIds.get(tableId), columnIdsInPredicates.get(tableId),
                 candidateIndexIdToMeta);
         // Step3: group by list in query is the subset of group by list in view or view contains no aggregation
-        checkGrouping(columnNameToIds.get(tableId), columnIdsInGrouping.get(tableId), candidateIndexIdToMeta);
+        checkGrouping(columnNameToIds.get(tableId), columnIdsInGrouping.get(tableId), candidateIndexIdToMeta,
+                disableSPJGMVs.get(tableId), isSPJQueries.get(tableId));
```

Comment:
```suggestion
                disableSPJGMVs.get(tableId), isSPJQueries.get(relationId));
```
Can be replace all tableId/scanId/table to relationId?